tests/all_tests_v2.nim

```diff
@@ -1,4 +1,5 @@
 import
   # Waku v2 tests
   ./v2/test_waku,
-  ./v2/test_wakunode
+  ./v2/test_wakunode,
```

Comment:
Can we expose this in the Makefile? Currently `make wakutest2` only tests `test_waku`. There is a Nim task `test2` that works, so we could change the wakutest2 task in waku.nimble to call `test "all_tests_v2"` instead.

Doing this locally
```task wakutest2, "Build Experimental Waku tests":
  test "all_tests_v2"
```
means `make wakutest2` runs all the relevant tests.

Reply:
yeah sure, I can do that in another PR.

Reply:
I mean, just add it to this PR, you added the tests...

Reply:
https://github.com/status-im/nim-waku/pull/107